autoload/neomake/makers/ft/cuda.vim

```diff
@@ -0,0 +1,12 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#cuda#EnabledMakers()
+    return ['nvcc']
+endfunction
+
+function! neomake#makers#ft#cuda#nvcc()
+    return {
+        \ 'exe': 'nvcc',
+        \ 'errorformat': '%f\(%l\):%m',
```

Comment:
There will be no error type assigned (error, warning, info etc).
Could this information be parsed from the output, e.g. with some special argument/format?